mlir/lib/Dialect/SparseTensor/Transforms/SparseAssembler.cpp

```diff
@@ -33,20 +33,14 @@ static void convTypes(TypeRange types, SmallVectorImpl<Type> &convTypes,
     }
     // Convert the external representation of the values array.
```

Comment:
This comment can be removed now

maybe move the single remaining sst L35 after comment at L36 to get a clean new section